### PR TITLE
MEP: CRI: Containerd by default

### PR DESCRIPTION
**Pull request by afbjorklund**
See #9639

### CODE REVIEW

File: enhancements/proposed/20201230-containerd-default/container-default.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+* Remove the docker support from minikube
+* Change anything in the docker driver
+
+## Design Details
+
+The containerd container runtime is already included, and is passing certification.
+
+Unlike Docker, will need to include the CRI (runtime) and CNI (network) by default.
+
+Use BuildKit as a complement to Containerd, for producing an image from Dockerfile.
```
Comment (medyagh) [inline enhancements/proposed/20201230-containerd-default/container-default.md:37]: add link to BuildKit to clarify which one you mean.

File: enhancements/proposed/20201230-containerd-default/container-default.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Please leave the above text in your proposal as instructions to the reader.
+
+## Summary
+
+Change default container runtime, from current "docker" to replacement "containerd".
+
+## Goals
+
+* Change from docker to containerd as default
+* Still allow building images with minikube
```
Comment (medyagh) [inline enhancements/proposed/20201230-containerd-default/container-default.md:24]: add still allow "fast image building" (comparable to docker-env method)

**Review by medyagh — CHANGES_REQUESTED** [review]

File: enhancements/proposed/20201230-containerd-default/container-default.md (hunk 1 above)
Comment (afbjorklund) [reply enhancements/proposed/20201230-containerd-default/container-default.md:37]: Ok

File: enhancements/proposed/20201230-containerd-default/container-default.md (hunk 2 above)
Comment (afbjorklund) [reply enhancements/proposed/20201230-containerd-default/container-default.md:24]: Well, docker-env only works with docker - but I guess you mean building on some other docker daemon and pushing.

Like how kind and microk8s are doing it, when there is no build support within the cluster. With a save|load (tar) pipe...

**Review by tstromberg — APPROVED** [review]

**Review by priyawadhwa — APPROVED** [review]

**Pull request merged**